addons/web/static/src/views/kanban/kanban_controller.xml

```diff
@@ -28,6 +28,19 @@
                     Create
                 </button>
             </t>
+            <t t-foreach="headerButtons" t-as="button" t-key="button.id">
+                <ListViewHeaderButton
```

Comment:
we should rename this into "ViewHeaderButton" or something, and move it out of list/

Reply:
I had forgotten to push the rename commit.